backend/apps/advanced_search/views.py

```
@@ -0,0 +1,20 @@
+    serializer_class = SearchEmbeddingSerializer
+    permission_classes = [IsAuthenticated]
+
+class UserSearchProfileViewSet(viewsets.ModelViewSet):
+    queryset = UserSearchProfile.objects.all()
+    serializer_class = UserSearchProfileSerializer
+    permission_classes = [IsAuthenticated]
+
+class SearchAnalyticsViewSet(viewsets.ModelViewSet):
+    queryset = SearchAnalytics.objects.all()
+    serializer_class = SearchAnalyticsSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)